docs/docs/examples/finetuning/embeddings/finetune_corpus_embedding.ipynb

```diff
@@ -0,0 +1,384 @@
+{
```

Comment:
<p>Could we explain this section a bit more?</p><p> </p><p>Why do we finetune both a model and an adaptor? Can we also print the eval results so that we can properly compare to a baseline?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/15954/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Added some more details - let me know if you think it needs more. The results comparing NUDGE, adapter fine-tuning, and no fine-tuning are logged at the end of the notebook

Reply:
Great! This seems way clearer now